week3/server.sql

```diff
@@ -0,0 +1,40 @@
+CREATE TABLE `customer` (
+  `ID` INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `f_name` varchar(35) NOT NULL,
```

Comment:
I would encourage you to use complete words when naming columns. f_name could mean full_name or first_name. For people who are using the database after you, it is always easier to understand when there are no abbreviations.